.github/workflows/test.yml

```diff
@@ -44,14 +44,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.19.x, 1.20.x]
+        go-version: [1.22.x, 1.23.x]
```

Comment:
We should still include 1.19 here for go.mod, right? Or do something clever with the `go-version-file:` support in setup-go?

Reply:
I'm not sure it makes sense to test on older Go releases, unless we explicitly want to maintain compatibility with older Go versions (and for now I don't know a reason to do that).

go version in `go.mod` will raise naturally once we'll start using newer features.

Reply:
Besides, we're switching to go 1.21 as a minimum in #777

Reply:
Would it be ok to use;

- `min` version (go1.19 or go1.21 after #777)
- `latest`
- `latest - 1`

The last 2 being the currently supported versions, and the `min` version to verify that the minimum version specified in `go.mod` isn't lying.

Reply:
Updated; PTAL @thaJeztah

Reply:
GitHub requires explicitly resolving conversations before merge; that's absolutely bonkers 🙃